guides/v15-mdc-migration.md

```diff
@@ -501,10 +501,75 @@ DOM and CSS of the components, you may need to tweak some of your application's
 * The label is closer to the enabled toggle
 
 ### Slider
-<!-- TODO(wagnermaciel): review -->
 
 * Sliders now work with mobile device screen readers.
 
+* The slider template API has changed from a single `<mat-slider>` element to a `<mat-slider>`
+  element which contains one or two `<input>` elements (depending on whether the slider should)
+  be a standard or range slider. E.g.
+  ```html
+    <!-- Single slider -->
+    <mat-slider>
+      <input matSliderThumb>
+    </mat-slider>
+
+    <!-- Range slider -->
+    <mat-slider>
+      <input matSliderStartThumb>
+      <input matSliderEndThumb>
+    </mat-slider>
+  ```
+
+* The new `discrete` property on the `<mat-slider>` now controls whether the slider has tick marks
+  and a value indicator tooltip. It replaces `thumbLabel`.
+
+  ```html
+  <!-- Before -->
+  <mat-slider thumbLabel></mat-slider>
+
+  <!-- After -->
+  <mat-slider discrete>
+    <input matSliderThumb>
+  </mat-slider>
+  ```
+  
+* The `tickInterval` property has been removed. To switch to the new API, use `showTickMarks` to
+  create a slider with tick marks, and the interval for your tick marks will match your slider's
+  `step`. The `tickInterval` property is under consideration to be added back in future releases.
+
+  ```html
+  <!-- Before -->
+  <mat-slider tickInterval="5" step="5"></mat-slider>
+
+  <!-- After -->
+  <mat-slider step="5" showTickMarks>
+    <input matSliderThumb>
+  </mat-slider>
+  ```
+
+* The `displayValue` property has been removed. The suggested alternative for controlling the
+  value indicator text is to provide a function via `displayWith`.
+
+  ```html
+  <!-- Before -->
+  <mat-slider [displayValue]="myDisplayValue"></mat-slider>
+
+  <!-- After -->
+  <mat-slider [displayWith]="myDisplayWithFn">
+    <input matSliderThumb>
+  </mat-slider>
+  ```
+
+* The `<input>` elements now control the following:
+    - `value`
+    - `percent`
+    - `valueText` is now replaced by the native input's aria-valuetext
+
+* The `invert` and `vertical` properties have been removed 
+
+* The slider API has also changed such that there is a new component: `MatSliderThumb`.
```

Comment:
this still doesn't make sense, there's nothing following "the following:"

Reply:
I think I managed to screw up my last commit. Sorry about that